Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11
+
+WORKDIR ./ 
+
+RUN pip install  poetry
```

Comment:
Stop trying to install poetry with pip. This will eventually break everything 😁

Reply:
can i use pipx ?

Reply:
@aalysher just use the poetry script.